lib/spotlight_search/utils.rb

```diff
@@ -2,7 +2,10 @@
 
 module SpotlightSearch::Utils
   class << self
-    def serialize_csv_columns(*columns, **hashes)
+    def serialize_csv_columns(*columns)
+      association_hash = columns.map{|hashes| hashes if hashes.class.eql?(Hash)}.compact.last
+      hashes = association_hash.nil? ? {} : association_hash
```

Comment:
if the variable is hashes, other side should be plural right?

why not  association_hash || {}